docs/getting-started/tutorial/run-the-project-pipeline/index.md

```diff
@@ -48,7 +48,7 @@ stored in the `output` folder.
 <ol>
```

Comment:
fbcc2caf4620237466f0c51f320ca91cd82a3d0b broke the rendering of backticked `code` segments within this HTML list. Not your doing, I'll take a look at fixing or we can work on this together if you're interested.